test/e2e_node/os_label_rename_test.go

```diff
@@ -0,0 +1,101 @@
+//go:build cgo && linux
+// +build cgo,linux
```

Comment:
curious why this test requires cgo ... cgo compilation of e2e_node tests is causing issues on go1.20 (xref https://github.com/kubernetes/kubernetes/pull/115613 and https://github.com/golang/go/issues/58425)

Reply:
I think it was a copy-paste error wrt cgo. I'll remove it an open a PR.